dist/src/main/java/io/camunda/application/MigrationApplication.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application;
+
+import io.camunda.application.commons.CommonsModuleConfiguration;
+import io.camunda.application.commons.service.CamundaServicesConfiguration;
+import io.camunda.application.commons.service.ServiceSecurityConfiguration;
+import io.camunda.application.listeners.ApplicationErrorListener;
+import io.camunda.migration.api.MigrationException;
+import io.camunda.migration.api.Migrator;
+import io.camunda.zeebe.gateway.GatewayModuleConfiguration;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadPoolExecutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.Profiles;
+
+@SpringBootApplication
+@Profile({"migration", "identity-migration | process-migration"})
```

Comment:
❓ Just for my understanding (call me spring noob): Does this mean 

A) `migration` AND one of `identity-migration | process-migration` has to be specified 
or 
B) any of the profiles will enable this

Reply:
If this is the Standalone application I guess we shouldn't have the profiles here.

Reply:
I think we should do it similar to what we have with the other Standalone apps for example look at the Broker

* We have[ a StandaloneBroker](https://github.com/camunda/camunda/blob/main/dist/src/main/java/io/camunda/application/StandaloneBroker.java#L15-L18) - with implementing the main etc
* We have[ a Broker module configuration - that is enabled by profile](https://github.com/camunda/camunda/blob/main/dist/src/main/java/io/camunda/zeebe/broker/BrokerModuleConfiguration.java#L36-L43)


Similar we have for gateway, operate,tasklist, etc.

Does this make sense @panagiotisgts ?

Reply:
- It should be one of the three, forgot to remove the `|` operator 
- - The `migration` profile, activates the other two as well
- - If none of these are present it will not run

Yeah, it makes sense, will fix this one.